scripts/fetch_secrets/fetch_secrets.rb

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env ruby
+require 'getoptlong'
+require 'logger'
+require 'yaml'
+require 'aws-sdk-ssm'
+require 'erb'
+require 'tempfile'
+
+ALLOWED_SOURCES = ['parameter', 'parameter-path', 'yaml-file']
```

Comment:
is it worth prefixing `aws-ssm` as we'll be introducting KeyVault?

Reply:
Changed, thanks